File: src/test/scala/com/infotarget/codility/scala/FrogRiverOneTest.scala
package com.infotarget.codility.scala

import org.scalatest.FunSuite

class FrogRiverOneTest extends FunSuite {

  test("testSolution") {
    assert(6 === FrogRiverOne.solution(5, Array[Int](1, 3, 1, 4, 2, 3, 5, 4)))
  }

}

File: src/test/java/com/infotarget/codility/java/FrogRiverOneTest.java
package com.infotarget.codility.java;

import org.junit.Assert;
import org.junit.Test;

public class FrogRiverOneTest {

    private FrogRiverOne frogRiverOne = new FrogRiverOne();

    @Test
    public void testSolution() throws Exception {
        Assert.assertEquals(6, frogRiverOne.solution(5, new int[]{1, 3, 1, 4, 2, 3, 5, 4}));
    }
}